### PR TITLE
Add livekit-plugins-google-adk for Google ADK integration

### PR DESCRIPTION
**Pull request by Speediing**
## Summary
- Adds `livekit-plugins-google-adk` plugin that wraps Google ADK agents as LiveKit LLM plugins
- `LLMAdapter` wraps any ADK `BaseAgent`/`LlmAgent` so it can be used in LiveKit voice pipelines — ADK handles tool calling and multi-agent orchestration internally, LiveKit manages voice orchestration (turns, interruptions, STT/TTS)
- Fixes a bug where the adapter called `ChatContext.messages()` (non-existent) — now correctly uses `ChatContext.items` with `ChatMessage` filtering
- 39 unit tests covering responses, streaming, error handling, session management, edge cases
- E2E test script (`tests/e2e_google_adk.py`) for manual verification against real Gemini API
- Example voice agent (`examples/voice_agents/google_adk_agent.py`)

## Test plan
- [x] 39/39 unit tests passing (`uv run pytest tests/test_google_adk.py -v`)
- [x] E2E smoke test passing with real Gemini API
- [x] E2E tool calling test passing (ADK invokes tools through the adapter)
- [x] E2E session reuse test passing (memory persists across turns)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: tests/test_google_adk.py [hunk 1]
```diff
@@ -0,0 +1,834 @@
...
+
+async def collect_chunks(stream) -> list:
+    """Collect all chunks from a stream."""
+    chunks = []
+    async for chunk in stream:
+        chunks.append(chunk)
+    return chunks
+
+
+async def collect_text_chunks(stream) -> list[str]:
```
Comment (davidzhao) [inline tests/test_google_adk.py:82]: we have a new function: [LLMStream.collect](https://github.com/livekit/agents/blob/main/livekit-agents/livekit/agents/llm/llm.py#L406)

File: tests/test_google_adk.py
Comment (davidzhao) [inline tests/test_google_adk.py:1]: could you add this to the list in makefile: make unit-tests?

File: livekit-plugins/livekit-plugins-google-adk/livekit/plugins/google_adk/llm.py [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    ADK handles tool calling and multi-agent orchestration internally.
+    LiveKit tools passed via ``chat()`` are not used — define tools on the
+    ADK agent instead.
+    """
+
+    def __init__(
+        self,
+        agent: LlmAgent | BaseAgent,
+        *,
+        runner: Runner | None = None,
```
Comment (davidzhao) [inline livekit-plugins/livekit-plugins-google-adk/livekit/plugins/google_adk/llm.py:61]: nit: slight preference for `NotGivenOr[Runner] = NOT_GIVEN` as a pattern, to allow future use to distinguish between "not set" vs "do not use"

**Review by davidzhao — COMMENTED** [review]
some minor comments. I would just keep this as part of the Google plugin, rather than creating a new one for ADK.. it could just be:

`livekit.plugins.google.adk.LLMAdapter`
